wolox-books/src/app/store/books.reducer.ts

```diff
@@ -0,0 +1,18 @@
+import { Action } from '@ngrx/store'
+import { Book } from 'app/models/book'
+import * as BookActions from './books.actions'
+
+export function reducer(state = [], action: BookActions.Actions) {
```

Comment:
I think you are using an outdated version of Ngrx. The reducer and actions have been changed for a new API, check it out 😄